### PR TITLE
Containerized tutorials

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: docker/tutorials/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Docker container with tutorial notebooks
+
+## Build container image
+
+Jupyter Docker images - https://github.com/jupyter/docker-stacks 
+
+```shell
+./docker/tutorials/build.sh
```
Comment (andre-merzky) [inline docker/tutorials/README.md:8]: I am probably missing a step before this - the command gives me:
```sh
$ ./docker/tutorials/build.sh
Sending build context to Docker daemon  314.5kB
Step 1/4 : FROM jupyter/minimal-notebook
Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io on [::1]:53: read udp [::1]:42162->[::1]:53: read: connection refused
```
any suggestion on what I need to set up?  Thanks!

Comment (mtitov) [reply docker/tutorials/README.md:8]: @andre-merzky Andre, it seems it is some connection issues on your side, I just re-run this and got expected output

<details>

```
SDK % ./docker/tutorials/build.sh
[+] Building 44.0s (9/9) FINISHED                                                                                                                                         
 => [internal] load build definition from Dockerfile                                                                                                                 0.0s
 => => transferring dockerfile: 133B                                                                                                                                 0.0s
 => [internal] load .dockerignore                                                                                                                                    0.0s
 => => transferring context: 2B                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/jupyter/minimal-notebook:latest                                                                                           1.0s
 => [auth] jupyter/minimal-notebook:pull token for registry-1.docker.io                                                                                              0.0s
 => [1/3] FROM docker.io/jupyter/minimal-notebook@sha256:4122f7a19251d308256c26f8c3d7cb436a9fc86017cf6f529df4c462fede2dfe                                           41.9s
 => => resolve docker.io/jupyter/minimal-notebook@sha256:4122f7a19251d308256c26f8c3d7cb436a9fc86017cf6f529df4c462fede2dfe                                            0.0s
 => => sha256:8a31808113566223808a87756c2fddf1c2c3200a50f247e3bab4a4961dd73a1e 11.45kB / 11.45kB                                                                     0.0s
 => => sha256:e96e057aae67380a4ddb16c337c5c3669d97fdff69ec537f02aa2cc30d814281 30.43MB / 30.43MB                                                                     4.4s
 => => sha256:4992ea90a334a21ed3b788b8c0869f56c3f639a4ecb263a5e30bceaec6c44e3d 680B / 680B                                                                           0.3s
 => => sha256:4122f7a19251d308256c26f8c3d7cb436a9fc86017cf6f529df4c462fede2dfe 772B / 772B                                                                           0.0s
 => => sha256:28d869151a1cefd7aa5cd12e613e1289337fc374c79fd911f4d04afeb0f50cba 3.86kB / 3.86kB                                                                       0.0s
 => => sha256:5601c95bdbfc5a3cb210f06ce351cdbcfd38c47b98484f61b745cad2e679d1ba 39.21MB / 39.21MB                                                                     5.7s
 => => sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1 32B / 32B                                                                             0.4s
 => => sha256:572f064378079e6a62d82054625d2807615db66185d9060d00c6d74207b4b28f 1.92kB / 1.92kB                                                                       0.5s
 => => sha256:976e7fddb10a8cfa42aeed235c72f8e82c33c8406122fd78fefc8e6bbdc46ee5 4.92kB / 4.92kB                                                                       0.5s
 => => sha256:65b0401fcda303f19c8596b19fc469c419f8934da6db7ede7441d725568a1744 149B / 149B                                                                           0.6s
 => => sha256:611f7b648b66e6182695b5611efa9705cc8ccd39c2487af8c44eb0673cf8e043 275B / 275B                                                                           0.7s
 => => sha256:01bbce505576c068d2e270c34ebf917c12ea71876c83d1d624dbc4d84097f5c4 208.88MB / 208.88MB                                                                  23.4s
 => => sha256:0a2d3aaa558a1287a95df7f5e03b94a3a04fde83604c130e07a311f20bcf8212 4.53kB / 4.53kB                                                                       4.5s
 => => extracting sha256:e96e057aae67380a4ddb16c337c5c3669d97fdff69ec537f02aa2cc30d814281                                                                            1.1s
 => => sha256:7e5c1881c57978f6df4a8e1fc4b46cba7c99c9cc186bc702068adb9f1ead73bb 1.08kB / 1.08kB                                                                       4.6s
 => => sha256:ebd1835c0a9e82756226185943b69b1c8e81a780301f58b4ae2d2e41f7becc19 1.16kB / 1.16kB                                                                       4.7s
 => => sha256:0139861f783b25105e94cc41cd0d31224d7b7b351286972ef424d49b067f5a46 177.83MB / 177.83MB                                                                  23.1s
 => => extracting sha256:5601c95bdbfc5a3cb210f06ce351cdbcfd38c47b98484f61b745cad2e679d1ba                                                                            1.8s
 => => sha256:b692686c2f27d5dec5abfb61141b232fb21e5a1e131377921d1e0334368f377f 1.46kB / 1.46kB                                                                       5.8s
 => => sha256:7bc2505592f97faff69ba54879fdd3218e389822d641bc337d69238070c4a949 434B / 434B                                                                           5.9s
 => => extracting sha256:4992ea90a334a21ed3b788b8c0869f56c3f639a4ecb263a5e30bceaec6c44e3d                                                                            0.0s
 => => extracting sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1                                                                            0.0s
 => => extracting sha256:572f064378079e6a62d82054625d2807615db66185d9060d00c6d74207b4b28f                                                                            0.0s
 => => extracting sha256:976e7fddb10a8cfa42aeed235c72f8e82c33c8406122fd78fefc8e6bbdc46ee5                                                                            0.0s
 => => extracting sha256:65b0401fcda303f19c8596b19fc469c419f8934da6db7ede7441d725568a1744                                                                            0.0s
 => => extracting sha256:611f7b648b66e6182695b5611efa9705cc8ccd39c2487af8c44eb0673cf8e043                                                                            0.0s
 => => extracting sha256:01bbce505576c068d2e270c34ebf917c12ea71876c83d1d624dbc4d84097f5c4                                                                            9.1s
 => => extracting sha256:0a2d3aaa558a1287a95df7f5e03b94a3a04fde83604c130e07a311f20bcf8212                                                                            0.0s
 => => extracting sha256:7e5c1881c57978f6df4a8e1fc4b46cba7c99c9cc186bc702068adb9f1ead73bb                                                                            0.0s
 => => extracting sha256:ebd1835c0a9e82756226185943b69b1c8e81a780301f58b4ae2d2e41f7becc19                                                                            0.0s
 => => extracting sha256:0139861f783b25105e94cc41cd0d31224d7b7b351286972ef424d49b067f5a46                                                                            8.4s
 => => extracting sha256:b692686c2f27d5dec5abfb61141b232fb21e5a1e131377921d1e0334368f377f                                                                            0.0s
 => => extracting sha256:7bc2505592f97faff69ba54879fdd3218e389822d641bc337d69238070c4a949                                                                            0.0s
 => [internal] load build context                                                                                                                                    0.0s
 => => transferring context: 271B                                                                                                                                    0.0s
 => [2/3] WORKDIR /tutorials/                                                                                                                                        1.0s
 => [3/3] COPY ./ ./                                                                                                                                                 0.0s
 => exporting to image                                                                                                                                               0.0s
 => => exporting layers                                                                                                                                              0.0s
 => => writing image sha256:a9caa3c172661d4a6367bba1bff48b2134f410336bd529847bc8887fb61b9f2a                                                                         0.0s
 => => naming to docker.io/exaworks/sdk-tutorials
```

</details>

**Comment by andre-merzky** [comment]
When testing this, the stumbles over the first cell of the rct notebook, the `pip install radical.entk`.  It fails with:
```
building 'netifaces' extension
      gcc -pthread -B /opt/conda/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /opt/conda/include -fPIC -O2 -isystem /opt/conda/include -fPIC -DNETIFACES_VERSION=0.11.0 -I/opt/conda/include/python3.10 -c netifaces.c -o build/temp.linux-x86_64-cpython-310/netifaces.o
      error: command 'gcc' failed: No such file or directory
      [end of output]
```

Any advise?

**Review by andre-merzky — APPROVED** [review]
LGTM, but I did not test-run.

**Comment by mtitov** [comment]
@andre-merzky I would like to work more on this PR, trying to incorporate your previous comments. Currently it provides a container with the possibility to run jupyter notebooks within it, but doesn't let to handle packages

**Comment by mtitov** [comment]
close this PR in favor of the current tutorials setup procedure (it doesn't bring anything new)

**Pull request closed without merging**